cvat/apps/annotation/annotation.py

```diff
@@ -77,13 +77,57 @@ def serialize(self):
         if serializer.is_valid(raise_exception=True):
             return serializer.data
 
+    @staticmethod
+    def _is_shape_inside(shape, start, stop):
+        return start <= int(shape['frame']) <= stop
```

Comment:
One tricky case here. Should we return True here for shapes with outside == True?

Reply:
added filter to remove starting shapes with outside==True after processing